tests/app/test_jinja_filters.py

```diff
@@ -253,10 +255,22 @@ def test_format_duration(duration, formatted_duration, app):
             {"maximum": {"value": 123456789012345678901123456789012345678901234567890}},
             None,
         ),
+        ({"maximum": {"value": {"identifier": "set-maximum", "source": "answers"}}}, 5),
+        (
+            {"minimum": {"value": {"identifier": "set-minimum", "source": "answers"}}},
+            15,
```

Comment:
Input field getting default 15 when no maximum is bad (when resolved minimum is 5), we should decide how to change the logic to avoid that, or is minimum and maximum always required?